bootstrap.php

```diff
@@ -130,9 +185,39 @@ public function htmlClosed()
   {
     $onboarding_markup = $this->getOnboardingLinkIfNoAppId();
     return <<<END
-
-  </form>$onboarding_markup
+              $onboarding_markup
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </section>
 </div>
+<script src="https://code.jquery.com/jquery-2.2.0.min.js"></script>
+<script type="text/javascript">
+  $('.intercom-oauth-cta').unbind('click').click(function() {
+    var url = 'https://app.intercom.io/oauth?client_id=08a72bc8-e2dc-4095-b877-c53bf0934c85&state=csrf';
+    var auth_window = window.open(url, 'Sign in', 'width=700,height=450');
+    var checkWindow = function() {
+      if (auth_window.closed) {
+        window.location = '/wp-admin/options-general.php?page=intercom';
+      } else if (window.oauth_success === undefined) {
+        setTimeout(checkWindow, 200);
+      }
+    };
+    checkWindow();
+  });
+  $('#intercom-secure-mode').unbind('click').click(function() {
+    $('#intercom-secure-mode').prop('checked', false);
+    if(confirm('Are you sure you want to enable secure mode for Intercom ?'))  {
+      window.location = '/wp-admin/options-general.php?page=intercom&enable_secure_mode=1';
```

Comment:
I'm actually not using this anymore I'll remove it. The auth is done in : https://github.com/intercom/wordpress-oauth/pull/1/files